package-lock.json

```diff
@@ -6642,15 +6643,15 @@
     "to-snake-case": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/to-snake-case/-/to-snake-case-1.0.0.tgz",
-      "integrity": "sha1-znRpE4l5RgGah+Yu366upMYIq4w=",
+      "integrity": "sha512-joRpzBAk1Bhi2eGEYBjukEWHOe/IvclOkiJl3DtA91jV6NwQ3MwXA4FHYeqk8BNp/D8bmi9tcNbRu/SozP0jbQ==",
```

Comment:
```
$ curl https://registry.npmjs.org/to-snake-case -s | jq '.versions."1.0.0".dist'
{
  "shasum": "ce746913897946019a87e62edfaeaea4c608ab8c",
  "tarball": "https://registry.npmjs.org/to-snake-case/-/to-snake-case-1.0.0.tgz"
}
```
^ npmjs does not serve a strong integrity hash.

Can you try running `dev-env$ bin/npm docstore ci` from docstore@master and `dev-env$ bin/npm docstore install @overleaf/metrics`?